pom.xml

```diff
@@ -496,7 +496,7 @@
 			<dependency>
 				<groupId>org.dbunit</groupId>
 				<artifactId>dbunit</artifactId>
-				<version>2.6.0</version>
+				<version>2.7.0</version>
```

Comment:
try out the changes with reference application

Reply:
thank @HerbertYiga I tested it in the same way as all the other PRs that I perform, by executing the jetty: run command, with the legacyUI module install.

Reply:
hello @dkayiwa @mozzy11 @sherrif10 @HerbertYiga  

I tested with the standalone reference application 
![Screenshot from 2020-06-16 11-01-27](https://user-images.githubusercontent.com/39851246/84761470-5c481c00-afc1-11ea-86eb-6b462f4b253f.png)